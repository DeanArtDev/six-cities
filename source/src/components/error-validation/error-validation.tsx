import * as React from 'react';

type Props = {
  errorContent: string;
}

export const ErrorValidation: React.FC<Props> = (props: Props) => {
  const {errorContent} = props;

  return (
    <section className={`error-validation`}>
      <h3 className={`visually-hidden`}>The error of login or password</h3>
      <p>{errorContent}</p>
    </section>
  );
};
