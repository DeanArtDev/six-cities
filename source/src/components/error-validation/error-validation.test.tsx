import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {ErrorValidation} from './error-validation';

describe(`<ErrorValidation/>: `, () => {

  it(`should render correctly`, () => {
    const tree = renderer
      .create(
          <ErrorValidation
            errorContent={`error`}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
