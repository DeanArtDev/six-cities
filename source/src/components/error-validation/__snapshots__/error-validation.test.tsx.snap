// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorValidation/>:  should render correctly 1`] = `
<section
  className="error-validation"
>
  <h3
    className="visually-hidden"
  >
    The error of login or password
  </h3>
  <p>
    error
  </p>
</section>
`;
